### DISCUSSION TITLE
How do i update / upgrade Keycloak?

### DISCUSSION THREAD

**Discussion by EndreEndi** [tteck/Proxmox#3805] [Help Needed] [answered]
Hey, umm, total noob here, i've installed Keycloak some time ago, and i don't see any way of updating / upgrading with the script or within the container, the only way i found is the official one from Keycloak which is kind of hard and i don't really understand it, so is there an easy way of updating / upgrading it?
I have Keycloak 25.0.2, and the latest is 25.0.6

**Comment by jfamiens** [comment, marked as answer]
go to the LXC  console - enter update
